.github/workflows/devnet-deploys.yml

```diff
@@ -165,6 +194,7 @@ jobs:
       - name: Deploy Aztec Nodes
         working-directory: ./yarn-project/aztec/terraform/node
         run: |
+          env
```

Comment:
This looks like a debug leftover but I don't think it's worth delaying the merge.